src/components/input-switch/foreignkey-field.tsx

```diff
@@ -83,9 +81,10 @@ const ForeignkeyField = (props: ForeignkeyFieldProps): JSX.Element => {
   const usedFormNumber = typeof props.formNumber === 'number' ? props.formNumber : 1;
 
   const { setValue, getValues } = useFormContext();
+  const { prefillAssociationSelectedRows } = useRecordedit();
```

Comment:
This should be prop that you pass to this component. We should try to avoid adding recordedit provider as a dependency of `InputSwitch` and its children. You can add it to the `foreignKeyCallbacks` prop.